apps/agreement/contracts/disputable/IDisputable.sol

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-License-Identitifer:    GPL-3.0-or-later
+ */
+
+pragma solidity 0.4.24;
```

Comment:
A bit of a nitpick, but usually for these inheritable contracts, we use license as MIT and allow a loose pragma.